README.md

```diff
@@ -6,13 +6,100 @@ via a dracut module.
 
 
 ## Kernel command line options for coreos-installer running in the initramfs
+
 * coreos.inst=yes - Instruct the installer to run
 * coreos.inst.install_dev - The block device on the system to install to
 * coreos.inst.image_url - The url of the coreos image to install to this device
 * coreos.inst.ignition_url - The url of the coreos ignition config (optional, enter
   coreos.inst.ignition_url=skip to not load an ignition config)
 
-## Testing out the installer script
+## Using the installer on Fedora or RHEL CoreOS
+
+This installer is incorporated into Fedora CoreOS and RHEL CoreOS.
+There are ISO images that can be downloaded that will allow for an
+install to be performed on bare metal hardware either via ISO install
```

Comment:
I'd phrase this in a way that suggests we like PXE more.

Reply:
Not necessarily trying to be too prescriptive here, but I will add a note about PXE probably being better for automation..